apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -272,6 +273,12 @@ export default function SectionsSetUpContainer({
   };
 
   const renderAdvancedSettings = () => {
+    // TODO: this will probably eventually be a setting on the course similar to hasTextToSpeech
+    // currently we're working towards piloting in Javalab in CSA only.
+    const aiTutorAvailable =
+      experiments.isEnabled('ai-tutor-toggle') &&
+      sections[0].course.displayName === 'Computer Science A';
```

Comment:
Could you double-check that `sections[0].course.displayName` is not a translatable element? 
I believe courses displayName is translated in some cases. It would cause problems if displayName returned a translated value.

Reply:
Good catch. Thanks!

Reply:
@mgc1194 I think this is important, but I might punt on it _for now_ since it will only affect a small group of pilot teachers and I only plan on doing a string comparison for while  I suss out which curriculum structure to attach `isAITutorAvailable` to.